### DISCUSSION TITLE
View subdomains in paraview

### DISCUSSION THREAD

**Discussion by rebpoli** [libMesh/libmesh#3017] [Q&A] [answered]
Hi,

I'm currently exporting Exodus files from libmesh into Paraview.
I could not find out how to export libmesh subdomains into a structure such that I can "show/hide" them in Paraview.

Is there any out-of-the-box solution for that?
If I want to see only fractures or only a single boundary_id, for example...

Thanks,
Renato

**Comment by jwpeterson** [comment, marked as answer]
LibMesh subdomains correspond to block IDs in paraview, and they already show up in a Paraview checklist which allows you to turn them of/off (see left-hand side of attached screenshot).

![subdomains](https://user-images.githubusercontent.com/1775907/133119744-d135bae1-7f9f-42ff-af80-49f2ffbe0f69.png)

The second part of your question refers to a `boundary_id`, and for that you can view specific sidesets in Paraview by first turning off (de-selecting) all the blocks, and then typing "sets" in the search box on the left-hand side menu of Paraview (see second screenshot) and selecting the sidesets you want to show/hide.

![sidesets_paraview](https://user-images.githubusercontent.com/1775907/133120149-cc5a669d-02bc-4c93-b787-bb86b9c05423.png)

**Comment by rebpoli** [comment]
Cool! I will try that.

Thanks,
Renato

Em seg., 13 de set. de 2021 13:19, John W. Peterson <
***@***.***> escreveu:
